device/mellanox/x86_64-mlnx_msn4700-r0/Mellanox-SN4700-V48C32/pg_profile_lookup.ini

```diff
@@ -0,0 +1,41 @@
+##
```

Comment:
Let's make this file a symbol link to `../Mellanox-SN4700-C128/pg_profile_lookup.ini`

Reply:
Will update

Reply:
Updated